AcademicManagerV1/src/com/academic/db/dao/CourseDAO.java

```diff
@@ -30,14 +32,18 @@ public CourseDAO(Connection conn) throws SQLException {
 				ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);
 		countStatement = dbConnection.prepareStatement("SELECT COUNT(*) FROM course WHERE isDeleted=0;",
 		ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);
+		
+		teachingStatement = dbConnection.prepareStatement(
+				"SELECT courseId,person.surname from teaching inner join person on teaching.personId = person.personId",
+				ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);
 
 		addStatement = dbConnection.prepareStatement(
 				"INSERT INTO course (courseId, title, cost, description, startingDate, endingDate, isActive, isDeleted) VALUES ('?','?','?','?','?','?','?','?')");
 
 		updateStatement = dbConnection.prepareStatement(
-				"UPDATE course SET title= ? , cost= ?, description= ?, startingDate= ?, endingDate= ? , isActive= ? WHERE courseId=?");
-
-		deleteStatement = dbConnection.prepareStatement("UPDATE course SET isDeleted=1 WHERE courseId=?");
+				"UPDATE course SET title=?,cost=?,description=?,startingDate=?,endingDate=?,isActive=? WHERE courseId= ?;");
+		
+		deleteStatement = dbConnection.prepareStatement("UPDATE course SET ,isDeleted=? WHERE courseId=?");
```

Comment:
Have a look at the latest code in the repository. This has been modified. This statement was wrong.